TestTicTacToe/GameRulesTest.cs

```diff
@@ -0,0 +1,219 @@
+using TicTacToe;
+using Xunit;
+
+namespace TestTicTacToe
+{
+    public class GameRulesTest
+    {
+
+        [Fact]
+        public void Should_Return_False_When_3_In_A_Row_Horizontally_Of_Different_Players_Marks()
+        {
+            // Arrange
+            var gameLosingBoard = new[,]
+            {
+                {1, 2, 1},
+                {0, 0, 0},
+                {0, 0, 0}
+            };
+        
+            // Act
+            var hasWonGame = GameRulesHandler.HasWon(gameLosingBoard);
```

Comment:
I think a better name here might be isGameWon, as the game can only be won when there are 3 of the same token in a row, otherwise it kinda sounds like you lose once three tokens have been placed